plugins/git/git.plugin.zsh

```diff
@@ -206,159 +129,213 @@ function ggu() {
 }
 compdef _git ggu=git-checkout
 
-alias ggpur='ggu'
-alias ggpull='git pull origin "$(git_current_branch)"'
-alias ggpush='git push origin "$(git_current_branch)"'
-
-alias ggsup='git branch --set-upstream-to=origin/$(git_current_branch)'
-alias gpsup='git push --set-upstream origin $(git_current_branch)'
-is-at-least 2.30 "$git_version" \
-  && alias gpsupf='git push --set-upstream origin $(git_current_branch) --force-with-lease --force-if-includes' \
-  || alias gpsupf='git push --set-upstream origin $(git_current_branch) --force-with-lease'
+function grename() {
+  if [[ -z "$1" || -z "$2" ]]; then
+    echo "Usage: $0 old_branch new_branch"
+    return 1
+  fi
 
-alias ghh='git help'
+  # Rename branch locally
+  git branch -m "$1" "$2"
+  # Rename branch in origin remote
+  if git push origin :"$1"; then
+    git push --set-upstream origin "$2"
+  fi
+}
 
-alias gignore='git update-index --assume-unchanged'
-alias gignored='git ls-files -v | grep "^[[:lower:]]"'
-alias git-svn-dcommit-push='git svn dcommit && git push github $(git_main_branch):svntrunk'
+#
+# Aliases
+# (sorted alphabetically)
+#
 
+alias glp='_git_log_prettily'
 alias gk='\gitk --all --branches &!'
 alias gke='\gitk --all $(git log --walk-reflogs --pretty=%h) &!'
```

Comment:
@carlosala 

There are this cases that don't match the order in the README, what do you think, leave them here or force tehm to match the README?

Reply:
I'd say match them, what do you think?

Reply:
Yes, match the README. It terms of order, the README becomes the source of truth.

Reply:
Let's go for it then 👌🏻